### ISSUE TITLE
128_shortcut1_inject1_none_2 in Jan 4, 2019?

### ISSUE THREAD

**Issue by chaipangpang** [LynnHo/AttGAN-Tensorflow#13]
Hi, there have some different between 128_shortcut1_inject1_none_2 pretrain model and 128_shortcut1_inject1_none pretrain model in google drive ?

**Comment by LynnHo** [comment]
@chaipangpang I increase the weight of the classification loss in 128_shortcut1_inject1_none_2.

**Comment by LynnHo** [comment]
@chaipangpang I made a mistake, I decreased the weight of reconstruction loss to 10 without modifying the classification rate.

**Comment by chaipangpang** [comment]
@LynnHo  g_loss = xb__loss_gan + xb__loss_att * 10.0 + xa__loss_rec * 10.0
Is that right?

**Comment by LynnHo** [comment]
> @LynnHo g_loss = xb__loss_gan + xb__loss_att * 10.0 + xa__loss_rec * 10.0
> Is that right?

@chaipangpang Yes.

**Comment by chaipangpang** [comment]
@LynnHo Thank you for your reply ,decreased the weight of reconstruction loss because model reconstruction is relatively easy?
I had try to train a AttGAN only edit glasses attribute(only one attribute), I have some discoveries :  
 1.reconstruct images have  better effect in 20 epoch;
 2.I used the default weight of every loss,but  the end result is not good for glasses attribute,I need modify weight of  loss or any other ?

**Comment by chaipangpang** [comment]
> > @LynnHo g_loss = xb__loss_gan + xb__loss_att * 10.0 + xa__loss_rec * 10.0
> > Is that right?
> 
> @chaipangpang Yes.

@LynnHo there is my sample_training and summaries in baidu yun https://pan.baidu.com/s/1Wi7-hjuv1Y872ribQ_RtUA (my google drive is so slow) , thank you.

**Comment by chaipangpang** [comment]
@LynnHo  Which parameter of model should be choosed, 'wgan',  'lsgan' or 'dcgan'?

**Comment by LynnHo** [comment]
@chaipangpang The default is 'wgan'. Our method does not perform well enough on 'Eyeglasses'.
